projects/packages/jetpack-mu-wpcom/src/features/verbum-comments/class-verbum-comments.php

```diff
@@ -584,8 +584,9 @@ public function should_load_gutenberg_comments() {
 	 * Check if we should show the subscription modal.
 	 */
 	public function should_show_subscription_modal() {
-		$modal_enabled = boolval( get_blog_option( $this->blog_id, 'jetpack_verbum_subscription_modal', true ) );
-		return ! is_user_member_of_blog( '', $this->blog_id ) && $modal_enabled;
+		$modal_enabled   = boolval( get_blog_option( $this->blog_id, 'jetpack_verbum_subscription_modal', true ) );
+		$is_jetpack_site = 522232 === get_current_blog_id();
+		return ! $is_jetpack_site && ! is_user_member_of_blog( '', $this->blog_id ) && $modal_enabled;
```

Comment:
If we want to check whether we're on a WordPress.com simple site, could we rely on the `Host` class, with `( new \Automattic\Jetpack\Status\Host() )->is_wpcom_simple()`?

I think it may also be worth adding a comment in there, explaining where the modal should be active and why.